test/fixtures/xml/usps/domestic_label_response.xml

```diff
@@ -0,0 +1,550 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<DelivConfirmCertifyV4.0Response>
+  <DeliveryConfirmationNumber>420902109400101699320000720205</DeliveryConfirmationNumber>
+  <DeliveryConfirmationLabel>JVBERi0xLjINCjUgMCBvYmoNCjw8DQovVHlwZSAvWE9iamVjdA0KL1N1YnR5cGUgL0ltYWdlDQov
```

Comment:
You don't really need the binary representations of the label images in the test - feel free to replace these with some kind of placeholder/marker, like `<LABEL IMAGE REDACTED DUE TO SIZE>`